package.json

```
@@ -151,7 +151,7 @@
     "@metamask/approval-controller": "^7.0.1",
     "@metamask/assets-controllers": "^30.0.0",
     "@metamask/base-controller": "^5.0.0",
-    "@metamask/composable-controller": "^3.0.0",
+    "@metamask/composable-controller": "npm:@metamask-previews/composable-controller@7.0.0-preview-1f29151",
```

This release is blocking this PR from being merged.

```suggestion
    "@metamask/composable-controller": "^8.0.0",
```